cri/v1alpha2/cri.go

```diff
@@ -76,10 +76,10 @@ const (
 	networkNotReadyReason = "NetworkPluginNotReady"
 
 	// passthruKey to specify whether a interface is passthru to qemu
-	passthruKey = "io.alibaba.pouch.runv.passthru"
+	passthruKey = "io.alibaba.pouch.vm.passthru"
```

Comment:
As an open source software, I don't think contain `alibaba` is a good idea, WDYT?
@flyer103 @Ace-Tang